srcpkgs/chrony/template

```diff
@@ -14,14 +14,12 @@ maintainer="Michal Vasilek <michal@vasilek.cz>"
 license="GPL-2.0-only"
 homepage="https://chrony.tuxfamily.org/"
 changelog="https://chrony.tuxfamily.org/news.html"
-distfiles="https://download.tuxfamily.org/chrony/${pkgname}-${version}.tar.gz"
-checksum=eafb07e6daf92b142200f478856dfed6efc9ea2d146eeded5edcb09b93127088
+distfiles="https://download.tuxfamily.org/chrony/chrony-${version}.tar.gz"
+checksum=19fe1d9f4664d445a69a96c71e8fdb60bcd8df24c73d1386e02287f7366ad422
 system_accounts="chrony"
 chrony_homedir="/var/lib/chrony"
 
-make_dirs="
-	/var/log/chrony 0755 chrony chrony
```

Comment:
`logdir /var/log/chrony` stopped being in the default config at some point, so I've removed this.